### ISSUE TITLE
Brand work planning

### ISSUE THREAD

**Issue by alex-page** [Shopify/polaris#140]
Plan for brand work based off of [cycle goals](https://docs.google.com/document/d/1pBsSBUtJXGOPiJmoLHRUDRX7S3S_FxBoMdzbx34oBII/edit)

Potential activities [here](https://docs.google.com/document/d/1LDQ_83lewFJXK6RkBL1sg2pB86FShWsSkYveL-y-lSU/edit#heading=h.rdy5lotaqrft)

- What outcomes do we want to achieve?
- What tasks / activities do we need to do to achieve those outcomes?

**Comment by sarahill** [comment]
After chatting with @jared-fanning @jamesbull and @johanstromqvist, we're thinking about not making the brand work a formal sprint for next week. We'll treat next week like more of a kick-off and focus on understanding the problems and gaining any additional context needed through learning.

The homesite will also likely influence the direction of the brand work and how we kick it off.

**Comment by alex-page** [comment]
Awesome @sarahill 💯

**Comment by jamesbull** [comment]
A few branding project resources to keep track of:
- [Project Brief: Launch foundational layout components](https://docs.google.com/document/d/1j4GQFYFob0WiQOjDJWIQccZxy281HX10qou4SfdXXQ8/edit#heading=h.ej5b62zdpznn)
- [Branding problem/opportunity](https://docs.google.com/document/d/15NwP8t5gi6zkvrSXt777odID3PfDNYD0yX0aJPFTKgs/edit#heading=h.is1qnkbt39h4)
- [Branding Sprint/Project Plan](https://docs.google.com/document/d/1LDQ_83lewFJXK6RkBL1sg2pB86FShWsSkYveL-y-lSU/edit#)
- [Q3C2 Cycle Project Planning Meeting Notes](https://docs.google.com/document/d/1pBsSBUtJXGOPiJmoLHRUDRX7S3S_FxBoMdzbx34oBII/edit#) - From Aug 4th

Some related docs/things to keep in mind:
- [Creating crafted layouts](https://docs.google.com/document/d/18ElvlkH8jE9FqPrfOnu-dOxlVnkcjveSlCHLeRNZc68/edit#heading=h.us2jv1bf4m61) - A proposal for working with partners across Shopify.
- [Rules of thumb for setting Polaris goals](https://docs.google.com/document/d/1x-9EXKGT78b16L_EqYrI5J6RWMz-DHt-DvC_YXLt8wk/edit) - Helpful guidance for goal setting from Yesenia and Kalv
-

**Comment by zecarlostorre** [comment]
Little update:

Week 1 is pretty much behind us, we changed [the plan](https://docs.google.com/document/d/1LDQ_83lewFJXK6RkBL1sg2pB86FShWsSkYveL-y-lSU/edit#) a bit along the way:

We moved the second part of the activity 3 (write a fake press release) to Activity 4 and scrapped the idea of righting a project brief, in favour of that fake press release.

That said, I think we made good progress and are in a good shape for week 2.
I added the plan for next week to the same doc (4 activities), Lo Kim is helping with scheduling it.
